### DISCUSSION TITLE
Packaging issues

### DISCUSSION THREAD

**Discussion by nubbix** [TomSchimansky/CustomTkinter#729] [Announcements]
Using pyinstaller or auto-py-to-exe after using customtkinter is not an easy task, I would stretch it as far as say impossible based on my knowledge. 

That said, I followed this example https://github.com/TomSchimansky/CustomTkinter/wiki/Packaging
and with that I made it into a directory, which btw produces a ton of files. Clicking then the exe inside, does not do anything. 

Also after trying out customtkinter, I seem to get this message in cmd for some reason.  Any idea there too?
![2022-11-21_220658](https://user-images.githubusercontent.com/6438354/203169558-891358b6-c5ae-47bc-bb4a-73f33a63ef0c.jpg)

Anyway, i digress.  I was wondering, will this be fixed so that we can down the line use one exe option as we can with regular tkinter?

**Comment by gcailly** [comment]
Could this help? 🙂
https://github.com/TomSchimansky/CustomTkinter/issues/631

**Comment by TomSchimansky** [comment]
I have no experience with using Tk commands directly through `root.eval()`. Maybe you should just remove this statement and ignore the window is not positioned in the center...
